common/parameters/Options.java

```diff
@@ -167,6 +169,18 @@ public Path reasonerDebuggerDir() {
         else throw TypeDBException.of(ILLEGAL_STATE);
     }
 
+    public long storageDataCacheSize() {
+        if (storageDataCacheSize != null) return storageDataCacheSize;
+        else if (parent != null) return parent.storageDataCacheSize();
+        else throw TypeDBException.of(ILLEGAL_STATE);
+    }
+
+    public long storageIndexCacheSize() {
+        if (storageIndexCacheSize != null) return storageIndexCacheSize;
+        else if (parent != null) return parent.storageIndexCacheSize();
+        else throw TypeDBException.of(ILLEGAL_STATE);
+    }
+
```

Comment:
new configurations the user can set: storage cache sizes for index and data blocks. These come from the Configuration